testlist.sh

```diff
@@ -1,20 +1,172 @@
+else 
+    test_array=("$@")  
+fi
+echo Tests to run are ${test_array[*]}
+
+# IOR test suite
+# Format:[<test name>]="min_servers max_servers num_targets num_ranks replication_type stonewall_timeout run/no_run"
+IOR_suite=( [16Clnt_IOReasy_SX]="1 256 16 32 SX 300 YES" [16Clnt_IORhard_SX]="1 256 16 32 SX 300 YES"
+            [16Clnt_IOReasy_2GX]="2 256 16 32 RP_2GX 300 NO" [16Clnt_IORhard_2GX]="2 256 16 32 RP_2GX 300 NO"
+            [16Clnt_IOReasy_3GX]="4 256 16 32 RP_3GX 300 NO" [16Clnt_IORhard_3GX]="4 256 16 32 RP_3GX 300 NO"
+            [1Sto4C_IOReasy_SX]="1 256 16 32 SX 60 NO" [1Sto4C_IORhard_SX]="1 256 16 32 SX 60 NO" 
+            [1Sto4C_IOReasy_2GX]="2 256 16 32 RP_2GX 60 NO" [1Sto4C_IORhard_2GX]="2 256 16 32 RP_2GX 60 NO"
+            [1Sto4C_IOReasy_3GX]="4 256 16 32 RP_3GX 60 NO" [1Sto4C_IORhard_3GX]="4 256 16 32 RP_3GX 60 NO" )
+
+# MDT test suite
+# Format:[<test name>]="min_servers max_servers num_targets num_ranks replication_type stonewall_timeout run/no_run"
+MDT_suite=( [16Clnt_MDTeasy_SX]="1 256 16 32 SX 300 NO" [16Clnt_MDThard_SX]="1 256 16 32 SX 300 NO"
+            [16Clnt_MDTeasy_2GX]="2 256 16 32 RP_2GX 300 NO" [16Clnt_MDThard_2GX]="2 256 16 32 RP_2GX 300 NO"
+            [16Clnt_MDTeasy_3GX]="4 256 16 32 RP_3GX 300 NO" [16Clnt_MDThard_3GX]="4 256 16 32 RP_3GX 300 NO"
+            [1Sto4C_MDTeasy_SX]="1 256 16 32 SX 60 NO" [1Sto4C_MDThard_SX]="1 256 16 32 SX 60 NO"
+            [1Sto4C_MDTeasy_2GX]="2 256 16 32 RP_2GX 60 NO" [1Sto4C_MDThard_2GX]="2 256 16 32 RP_2GX 60 NO"
+            [1Sto4C_MDTeasy_3GX]="4 256 16 32 RP_3GX 60 NO" [1Sto4C_MDThard_3GX]="4 256 16 32 RP_3GX 60 NO" )
+
+# Set the needed PATH and LD_LIBRARY_PATH
 export PATH=/opt/apps/xalt/xalt/bin:/opt/apps/intel19/python3/3.7.0/bin:/opt/apps/cmake/3.16.1/bin:/opt/apps/autotools/1.2/bin:/opt/apps/git/2.24.1/bin:/opt/intel/compilers_and_libraries_2019.5.281/linux/bin/intel64:/opt/apps/gcc/8.3.0/bin:/usr/lib64/qt-3.3/bin:/usr/local/bin:/bin:/usr/bin:/opt/ibutils/bin:/opt/ddn/ime/bin:/opt/dell/srvadmin/bin:.
 export LD_LIBRARY_PATH=/opt/apps/intel19/python3/3.7.0/lib:/opt/intel/debugger_2019/libipt/intel64/lib:/opt/intel/compilers_and_libraries_2019.5.281/linux/daal/lib/intel64_lin:/opt/intel/compilers_and_libraries_2019.5.281/linux/tbb/lib/intel64_lin/gcc4.7:/opt/intel/compilers_and_libraries_2019.5.281/linux/mkl/lib/intel64_lin:/opt/intel/compilers_and_libraries_2019.5.281/linux/ipp/lib/intel64:/opt/intel/compilers_and_libraries_2019.5.281/linux/compiler/lib/intel64_lin:/opt/apps/gcc/8.3.0/lib64:/opt/apps/gcc/8.3.0/lib:/usr/lib64/:/usr/lib64/
 
-DAOS_DIR="<path_to_daos>"
-DST_DIR="<path_to_daos_scaled_testing>"
-RES_DIR="<path_to_result_dir>"
-JOBNAME="<sbatch_jobname>"
-TIMEOUT="<sbatch_timeout>" #<hh:mm:ss>
-EMAIL="<email>" #<first.last@intel.com>
+# Define the location od daos, sacled testing, results. Extract the build identity as work week identity
+DAOS_DIR="/work/06758/arunar/frontera/BUILDS/20WW33.3+eccd948/daos"
```

Comment:
Do not hardcode your path here.
Move all user config var to the top so user don't have to search for it.